node-manager/src/nodemanager.rs

```diff
@@ -17,10 +22,23 @@ use crate::{
 pub struct NodeManager {
     mnemonic: Mnemonic,
     wallet: MutinyWallet,
+    node_storage: Mutex<NodeStorage>,
     ws_write: Arc<Mutex<SplitSink<WebSocket, Message>>>,
     counter: usize,
 }
 
+#[derive(Serialize, Deserialize, Clone)]
+pub struct NodeStorage {
+    pub nodes: Vec<NodeIndex>,
+}
+
+#[derive(Serialize, Deserialize, Clone)]
+pub struct NodeIndex {
+    pub id: String,
+    pub pubkey: String,
+    pub child_index: i32,
```

Comment:
I dont think this will ever be negative so we can use u32 here

Reply:
done